gcovr/templates/style.css

```diff
@@ -1,19 +1,26 @@
 :root {
     font-family: sans-serif;
-    --unknown_color: {{unknown_color}};
-    --low_color: {{low_color}};
-    --medium_color: {{medium_color}};
-    --high_color: {{high_color}};
-    --covered_color: {{covered_color}};
-    --uncovered_color: {{uncovered_color}};
-    --excluded_color: {{excluded_color}};
-    --warning_color: {{warning_color}};
-    --takenBranch_color: {{takenBranch_color}};
-    --notTakenBranch_color: {{notTakenBranch_color}};
-    --takenDecision_color: {{takenDecision_color}};
-    --uncheckedDecision_color: {{uncheckedDecision_color}};
-    --notTakenDecision_color: {{notTakenDecision_color}}
+    {# These are the default colors for the "green" theme #}
+    --unknown_color: lightgray;
+    --low_color: #FF6666;
+    --medium_color: #F9FD63;
+    --high_color: #85E485;
+    --covered_color: #85E485;
+    --uncovered_color: #FF8C8C;
+    --excluded_color: #53BFFD;
+    --warning_color: orangered;
+    --takenBranch_color: green;
+    --notTakenBranch_color: red;
+    --takenDecision_color: green;
+    --uncheckedDecision_color: darkorange;
+    --notTakenDecision_color: red;
```

Comment:
Shall we move this to a `:root.theme-green` and the common parts to a `:root.theme-green :root.theme-blue` to get a report with no colors if the theme isn't green or blue? Or better rename the theme `green` to `default`?

Reply:
I think its sensible to provide some colors by default. This makes it possible for the blue theme to just override a couple of variables. 

We could rename the `green` theme, but should it then only be renamed in CSS or also on the command line? Because changing the CLI argument would break backwards compatibility, whereas using different names in different parts of the program would be confusing. So I'd tend to leave all names as they currently are.

It would of course be possible to only add `theme-X` classes for non-default themes, but I think it's clearer to mention the selected theme, even if it doesn't do anything right now.

Reply:
You're right, since `green` is a CLI argument it shouldn't be renamed. Following CSS is working in Firefox and Safari and seems more clear for me:
```css
:root {
    font-family: sans-serif;
    --tab_size: 4;
}

:root.theme-green, :root.theme-blue {
    --unknown_color: lightgray;
    --low_color: #FF6666;
    --medium_color: #F9FD63;
    --high_color: #85E485;
    --covered_color: #85E485;
    --uncovered_color: #FF8C8C;
    --excluded_color: #53BFFD;
    --warning_color: orangered;
    --takenBranch_color: green;
    --notTakenBranch_color: red;
    --takenDecision_color: green;
    --uncheckedDecision_color: darkorange;
    --notTakenDecision_color: red;
}

:root.theme-blue {
    --high_color: #66B4FF;
    --covered_color: #66B4Ff;
    --takenBranch_color: blue;
}
```

Reply:
Or move the theme specific colors in a separate block for each theme.

Reply:
yes that looks a lot clearer